### PR TITLE
feat(typescript-lcfirst): lowercase the first characters of the text

### PR DESCRIPTION
**Pull request by ddouble-d**
- Programming langugage: TypeScript
- Function(s) that I'm implementing: lcfirst
- Issue number: #18 

- [x] I have created unit tests to support my implementation
- [x] I have created in-code documentation (or docstring) to document my code
- [ ] (Optional) I have updated the README.md file to document the function I'm implementing

### CODE REVIEW

**Review by aldy505 — APPROVED** [review]

**Comment by aldy505** [comment]
ada merge conflict

File: typescript/src/string/lcfirst.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const lcfirst = (text: string): string => text.charAt(0).toLowerCase() + text.slice(1);
```
Comment (elianiva) [inline typescript/src/string/lcfirst.ts:1]: tambahin docstring, refer ke dokumentasi PHP buat susunan kalimatnya

File: typescript/src/string/lcfirst.test.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  });
+
+  it("should return lowercased first characters from all uppercased text", () => {
+    expect(lcfirst(text.toUpperCase())).toEqual("sHIZUMU YOU NI TOKETE YUKU YOU NI.");
+  });
+
+  it("should return lowercased first characters from all lowercased text", () => {
+    expect(lcfirst(text.toLowerCase())).toEqual("shizumu you ni tokete yuku you ni.");
+  });
+});
```
Comment (elianiva) [inline typescript/src/string/lcfirst.test.ts:17]: tambahin test case buat invalid parameter, misal null, undefined, empty string, dll

**Review by elianiva — CHANGES_REQUESTED** [review]

File: typescript/src/string/lcfirst.ts (hunk 1 above)
Comment (ddouble-d) [reply typescript/src/string/lcfirst.ts:6]: done in [295b088](https://github.com/teknologi-umum/pehape/pull/65/commits/295b08812b93349f40d53bd6a9b7d16c3d644548)

File: typescript/src/string/lcfirst.test.ts (hunk 2 above)
Comment (ddouble-d) [reply typescript/src/string/lcfirst.test.ts:42]: done in [6cdf8f3](https://github.com/teknologi-umum/pehape/pull/65/commits/6cdf8f30960242c41d2cfd0adcd4937bce29cc1e)

**Comment by ddouble-d** [comment]
hi @elianiva 
can you re-review if you don't mind?

**Review by elianiva — APPROVED** [review]

**Pull request merged**